lib/engine/round/operating.rb

```diff
@@ -71,6 +71,10 @@ def recalculate_order
       def teleported?(entity)
         entity.abilities(:teleport)&.find(&:used?)
       end
+
+      def calculate_route_bonus
```

Comment:
i don't think we need a route bonus

Reply:
I will try to use hex_bonus (which I have renamed to hexes_bonus) for both this cases.